backend/service/chaos/experimentation/experimentstore/storer.go

```diff
@@ -52,10 +52,11 @@ func New(_ *any.Any, logger *zap.Logger, _ tally.Scope) (service.Service, error)
 		return nil, errors.New("experiment store wrong type")
 	}
 
-	transformer := NewTransformer()
+	sugaredLogger := logger.Sugar()
```

Comment:
we dont use a `sugaredlogger` anywhere else in clutch, just curious why were using it here?

Reply:
I used it because server experimentation used it - I did not realize that other packages were using a non logger. I'm going to address this in a separate PR I think. Thanks for pointing this one out